www/md_base/src/app/common/services/buildbot/buildbot.service.coffee

```diff
@@ -0,0 +1,203 @@
+class BuildbotService extends Factory
```

Comment:
It seems that gulp cannot follow symlinks for folders when compiling, unless we make symlinks for the files one by one. It makes the things a little complex.